submissions/maryOkosun/javascript/loop.js

```diff
@@ -0,0 +1,14 @@
+for (var i = 1; i < 116; i++) {
```

Comment:
You need to perform the difference between 7 and 123 and return the absolute value

Reply:
Thanks. Made the necessary correction. Please review my PR